rasgotransforms/rasgotransforms/transforms/drop_columns/drop_columns.sql

```diff
@@ -12,8 +12,13 @@ FROM {{source_table}}
 
 {%- if exclude_cols is defined -%}
 {%- set source_col_names = get_columns(source_table) -%}
-{%- set new_columns = source_col_names | reject('in', exclude_cols) -%}
 
+    {%- if not case_sensitive_exclude -%}
```

Comment:
Could we write this in a way that defaults to True? That way users only need to pass the param if they want to turn ON case sensitivity (which we'd be betting is the less frequently occurring case)

Reply:
I agree with this and I think the way it is written it is case insensitive by default which is what we want, but the name might make more sense if it's just 'case_sensitive'.